### PR TITLE
Fix wrong order of code blocks

### LINKED ISSUE #231: Wrong order of text in the Getting Help section of the "Get Started with Podman" page
The text seems to be out of order and does not match the following screenshot:

screenshot

URL: <https://podman.io/get-started>

### PR DESCRIPTION
**Pull request by GabiNecasova**
Fixes: #231

### CODE REVIEW

**Comment by lsm5** [comment]
@GabiNecasova the PR needs signoff:

```
$ git commit --amend -s
# Save it
$ git push -f
```

**Review by lsm5 — APPROVED** [review]
LGTM

@TomSweeneyRedHat PTAL
cc @brlin-tw

File: src/pages/get-started.tsx [hunk 1]
```diff
@@ -26,19 +26,20 @@ const GetHelpSection = () => {
...
             <p className="max-w-sm text-white">
               To get some help and find out how Podman is working, you can use the help.
             </p>
             {/* prettier-ignore */}
             <CodeBlock language="bash" showLineNumbers>
-                $ man podman {'\n'} 
-                $ man podman subcommand {'\n'}
+                $ podman --help # get a list of all commands {'\n'}
+                $ podman subcommand --help # get info on a command{' '}
+                {'\n'}
```
Comment (TomSweeneyRedHat) [inline src/pages/get-started.tsx]: I think, but don't know, that this has to be at the end fo the previous line.  @cgiradkar  do you know?

**Comment by TomSweeneyRedHat** [comment]
The rearranging and verbiage LGTM.  I'm just not sure about the carriage return before the `\n` block.

File: src/pages/get-started.tsx (hunk 1 above)
Comment (GabiNecasova) [reply src/pages/get-started.tsx]: @TomSweeneyRedHat  I tried to modify the code as: 
```
$ podman --help # get a list of all commands {'\n'}
$ podman subcommand --help # get info on a command{'\n'}
```

and the result is the same (please see the screenshot below). I tried to build it locally (`yarn build  & yarn serve`). 
Screenshot from 2023-11-06 11-17-21

Comment (cgiradkar) [inline src/pages/get-started.tsx]:
```suggestion
                $ podman subcommand --help # get info on a command
```

Comment (cgiradkar) [reply src/pages/get-started.tsx]: > I think, but don't know, that this has to be at the end fo the previous line. @cgiradkar do you know?

I agree, the extra carriage returns are not needed hence should be removed.

File: src/pages/get-started.tsx [hunk 2]
```diff
@@ -26,19 +26,20 @@ const GetHelpSection = () => {
             </p>
             {/* prettier-ignore */}
             <CodeBlock language="bash" showLineNumbers>
-                $ podman --help # get a list of all commands {'\n'}
-                $ podman subcommand --help # get info on a command{' '}
-                {'\n'}
+                $ man podman {'\n'} 
+                $ man podman subcommand {'\n'}
```
Comment (cgiradkar) [inline src/pages/get-started.tsx]:
```suggestion
                $ man podman subcommand
```

**Comment by GabiNecasova** [comment]
I removed both unnecessary newlines in the codeblocks. Should be fine now.

File: src/pages/get-started.tsx [hunk 3]
```diff
@@ -26,19 +26,19 @@ const GetHelpSection = () => {
             </p>
             {/* prettier-ignore */}
             <CodeBlock language="bash" showLineNumbers>
-                $ podman --help # get a list of all commands {'\n'}
-                $ podman subcommand --help # get info on a command{' '}
-                {'\n'}
+                $ man podman {'\n'} 
+                $ man podman subcommand
               </CodeBlock>
+
```
Comment (cgiradkar) [inline src/pages/get-started.tsx]:
```suggestion
```

**Comment by cgiradkar** [comment]
> I removed both unnecessary newlines in the codeblocks. Should be fine now.

There's another blank line introduced, plz clear that and squash and commit into 1 commit and force push

**Comment by GabiNecasova** [comment]
@cgiradkar Thank you for your note! I squashed commits as requested.

**Comment by cgiradkar** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
Welcome to the Podman.io community @GabiNecasova !

**Comment by brlin-tw** [comment]
I can verify that the defect is indeed fixed, thanks @GabiNecasova !

**Comment by GabiNecasova** [comment]
@brlin-tw  My pleasure :slightly_smiling_face:

**Pull request merged**